### ISSUE TITLE
Missing second elapsed time report

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#77]
```
$ cat tstim.pl

run :-
    time(tstim(_));
    time(tstim(_,_)),
    !, fail.

tstim(_)   :- !, fail.
tstim(_,_) :- true.
```
```
$ BProlog/bp -i tstim.pl
| ?- run.
CPU time 0.0 seconds.
CPU time 0.0 seconds.
no

$ swipl tstim.pl
?- run.
% 3 inferences, 0.000 CPU in 0.000 seconds (89% CPU, 564759 Lips)
% 1 inferences, 0.000 CPU in 0.000 seconds (92% CPU, 228519 Lips)
false.

# v2.5.25
$ tpl tstim.pl
?- run.
   % Time elapsed 2e-06s
   false.

```

**Comment by infradig** [comment]
Yes, it only reports time on success.

**Comment by infradig** [comment]
I'll  look into it.

**Comment by flexoron** [comment]
v2.5.31
```
$ tpl tstim.pl 
?- run.
   % Time elapsed 4e-06s
   % Time elapsed 1e-06s
   false.

?- length(_,1000000).
   true.

?- time(length(_,1000000)).
   % Time elapsed 0.367s
   true
;  % Time elapsed 0.0252s   % unexpected
   false.
?-
```

**Comment by infradig** [comment]
Oops.

**Comment by flexoron** [comment]
```
# v8.4.0
$ swipl
?- time(time((true;true))).
% -1 inferences, 0.000 CPU in 0.000 seconds (101% CPU, -153280 Lips)
% 134 inferences, 0.000 CPU in 0.000 seconds (98% CPU, 1523472 Lips)
true ;
% 5 inferences, 0.000 CPU in 0.000 seconds (101% CPU, 1150219 Lips)
% 137 inferences, 0.000 CPU in 0.000 seconds (98% CPU, 2230326 Lips)
true.
?- halt.

# v2.6.0
$ tpl
?- time(time((true;true))).
   % Time elapsed 1.3e-05s
   % Time elapsed 2.4e-05s
   true
;  % Time elapsed 1.5e-05s
   % Time elapsed 2.7e-05s
   true
;  % Time elapsed 0.000152s  % unreal?
   false.
?- halt.
```

**Comment by flexoron** [comment]
Thank you, fixed with v2.6.2.
